fdb-record-layer-lucene/src/main/java/com/apple/foundationdb/record/lucene/codec/LuceneOptimizedCompoundReader.java

```diff
@@ -138,61 +136,29 @@ public IndexInput openInput(String name, IOContext context) throws IOException {
         return res;
     }
 
-    /** Not implemented.
-     * @throws UnsupportedOperationException always: not supported by CFS */
-    @Override
-    public void deleteFile(String name) {
-        throw new UnsupportedOperationException();
-    }
-
-    /** Not implemented.
-     * @throws UnsupportedOperationException always: not supported by CFS */
-    @Override
-    public void rename(String from, String to) {
-        throw new UnsupportedOperationException();
-    }
-
-    /** Not implemented. */
-    @Override
-    public void syncMetaData() {
-        // Not implemented
-    }
-
     /** Returns the length of a file in the directory.
      * @throws IOException if the file does not exist */
     @Override
     public long fileLength(String name) throws IOException {
         ensureOpen();
         FileEntry e = entries.get(IndexFileNames.stripSegmentName(name));
-        if (e == null) {
+        if (e == null)
             throw new FileNotFoundException(name);
-        }
```

Comment:
We generally require braces, even for single line blocks like this. There's a checkstyle check, `NeedBraces`, that's supposed to catch that. It looks like the build failed (during javadoc generation) before it got to `fdb-record-layer-lucene:checkstyleMain`, so it may or may not

Reply:
Let me try again with Aaron's changes to the downloads...